### ISSUE TITLE
[Bug]: ImportError: cannot import name 'Function' from 'pymilvus'

### ISSUE THREAD

**Issue by ChengRuiLiang** [milvus-io/pymilvus#2312]
### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Describe the bug

Function and FunctionType are not found in pymilvus 2.4.4/2.4.8
![1729842220596](https://github.com/user-attachments/assets/039d9761-49b1-40ee-b24d-9341fff377bc)

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
- Method of installation (Docker, or from source):
- Milvus version (v0.3.1, or v0.4.0):
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by ChengRuiLiang** [comment]
![Uploading 1729843666247.jpg…]()

**Comment by ChengRuiLiang** [comment]
![1729843666247](https://github.com/user-attachments/assets/7083e4b3-f0d8-4a98-a8fa-99c0571c025b)

**Comment by XuanYang-cn** [comment]
@ChengRuiLiang `Function` will be available on pymilvus==2.5.0. You could clone PyMilvus and build it locally if you want to use it now.

**Comment by XuanYang-cn** [comment]
Should be fixed, PyMIlvus 2.5.0 released already
